shuffle/cli/src/deploy.rs

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{anyhow, Context, Result};
+use anyhow::{bail, format_err};
+use diem_config::config::NodeConfig;
+use diem_crypto::PrivateKey;
+use diem_sdk::{
+    client::BlockingClient,
+    transaction_builder::{Currency, TransactionFactory},
+    types::{
+        account_address::AccountAddress,
+        transaction::{Module, TransactionPayload},
+        LocalAccount,
+    },
+};
+use diem_temppath::TempPath;
+use diem_types::{
+    account_config, account_state::AccountState, account_state_blob::AccountStateBlob,
+    chain_id::ChainId, transaction::authenticator::AuthenticationKey,
+};
+use generate_key::load_key;
```

Comment:
Why didn't u have to write this code "use diem_config::config::generate_key::load_key" here?

Reply:
By importing `generate-key = { path = "../../config/generate-key" }` in cargo.toml in cli directory